meta-networking/recipes-support/strongswan/strongswan_5.6.2.bb

```diff
@@ -24,11 +24,14 @@ EXTRA_OECONF += "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '--with-syst
 
 
 PACKAGECONFIG ?= "charon curl gmp openssl stroke sqlite3 \
+        eap-identity eap-mschapv2 \
         ${@bb.utils.filter('DISTRO_FEATURES', 'ldap', d)} \
 "
 PACKAGECONFIG[aesni] = "--enable-aesni,--disable-aesni,,${PN}-plugin-aesni"
 PACKAGECONFIG[charon] = "--enable-charon,--disable-charon,"
 PACKAGECONFIG[curl] = "--enable-curl,--disable-curl,curl,${PN}-plugin-curl"
+PACKAGECONFIG[eap-identity] = "--enable-eap-identity,--disable-eap-identity,,${PN}-plugin-eap-identity"
+PACKAGECONFIG[eap-mschapv2] = "--enable-eap-mschapv2,--disable-eap-mschapv2,,${PN}-plugin-eap-mschapv2"
```

Comment:
Since the settings are already upstream, could you instead cherry-pick lines 33-34 in this layer and have an independent meta-nilrt commit which adds the `PACKAGECONFIG` words (L27) in our distro only?

That way, when we rebase the meta-oe layer, the settings lines will drop away cleanly.

Reply:
Done.